lib/streamlit/runtime/state/query_params_proxy.py

```diff
@@ -20,7 +20,8 @@
 
 
 class QueryParamsProxy(MutableMapping[str, str]):
-    """A stateless singleton that proxies `st.query_params` interactions
+    """
+    A stateless singleton that proxies ``st.query_params`` interactions
```

Comment:
nit: Is this docstring shown somewhere on the docs? If so, this is a quite technical/internal explanation and might needs something more simple.

Reply:
It is not (for now at least).